src/pyvesync/vesyncfan.py

```diff
@@ -1107,7 +1119,8 @@ def __build_api_dict(self, method: str) -> Tuple[Dict, Dict]:
         """
         modes = ['getHumidifierStatus', 'setAutomaticStop',
                  'setSwitch', 'setNightLightBrightness', 'setVirtualLevel',
-                 'setTargetHumidity', 'setHumidityMode', 'setDisplay']
+                 'setTargetHumidity', 'setHumidityMode', 'setDisplay',
+                 'setLevel']
```

Comment:
Just realized this will break the current 300s api since setLevel is not part of the response. Just add the feature logic for now and I will get this build out. Will have to restructure in the future